src/attributes/actions.ts

```diff
@@ -10,3 +15,13 @@ export const decrementFitness = (fitnessProgress: number): AppActions => ({
   type: DECREMENT_FITNESS,
   fitnessProgress
 });
+
+export const incrementCombat = (combatProgress: number): AppActions => ({
+  type: INCREMENT_COMBAT,
+  combatProgress
+});
+
+export const decrementCombat = (combatProgress: number): AppActions => ({
+  type: DECREMENT_COMBAT,
```

Comment:
could probably consolidate these actions into `incrementAttributeProgress(progress: number, attribute: Attribute)` . saves us making new actions for every one